packages/shared/comms/voice-over-comms.ts

```diff
@@ -13,6 +13,8 @@ import { put } from 'redux-saga/effects'
 import { setupPeer } from './peers'
 import { shouldPlayVoice } from './voice-selectors'
 import { positionObservable, PositionReport } from 'shared/world/positionThings'
+import { getUnityInstance } from 'unity-interface/IUnityInterface'
```

Comment:
I hesitate to put the `getUnityInstance` here in the comms package. @gonpombo8 what do you think?